style.css

```diff
@@ -0,0 +1,990 @@
+    width: 90%;
+    margin-bottom: 30px;
+  }
+
+  .character {
+    height: 48px;
+    justify-content: center;
+    align-items: center;
+    padding-top: 2px;
+    flex: .44;
+  }
+
+  .character__item_icon-left { width: 31%; }
+
+  .character_left { padding-left: 4px; }
+
+  .character_right { padding-right: 7px; }
+
+  .section-main__item_btns { width: 95%; }
+
+  .button__item_icon-main { width: auto; }
+
+  .button__item_alt-right {
+    height: 37px;
+    padding-right: 12px;
+    flex: .74;
+  }
+}
+
+/* Адаптив второй секции */
+
+/* Desktop */
+@media (max-width: 1150px) {
+  .section-desc { height: 780px; }
+
+  .section-desc__wrap_up { padding-top: 0;}
+
+  .text__wrap_main {
+    margin-bottom: 35px;
+    padding-left: 30px;
+  }
+
+  .text__wrap_title { padding-top: 40px; }
+
+  .text__wrap_character {
+    width: 85%;
+    height: 230px;
+    margin-bottom: 120px;
+    padding-left: 26px;
+  }
+}
+
+@media (max-width: 1030px) {
+  .text__wrap_main { padding-left: 20px; }
+
+  .text__content,
+  .text__wrap_character { width: auto; }
+
+  .text__wrap_character {
+    margin-bottom: 100px;
+    padding-left: 17px;
+  }
+}
+
+@media (max-width: 990px) {
+  .section-desc_future-drive { height: 870px; }
+
+  .text__title { padding-right: 0; }
+
+  .text__content { letter-spacing: normal; }
+  
+  .text__item_small-screen,
+  .text__item_small-wifi,
+  .text__item_small-bag { margin-right: 5px; }
+
+  .text__item_icon { max-width: 75%; }
+
+  .button__item_icon { margin-right: 3px; }
+
+  .button__item_big-order { padding-left: 11px; }
+}
+
+@media (max-width: 860px) {
+  .text__wrap_main,
+  .text__wrap_character { padding-left: 9px; }
+}
+
+/* Планшет */
+@media (max-width: 768px) {
+  .section-desc {
+    display: block;
+    width: auto;
+    height: auto;
+    background: 0;
+  }
+
+  .section-desc__wrap_up {
+    height: 486px;
+    padding-top: 45px;
+    background: url(./Project/item_interior.svg) no-repeat;
+    background-size: contain;
+    background-position: 0 100%;
+  }
+
+  .section-desc__title {
+    width: 94%;
+    text-align: right;
+    padding-left: 0;
+  }
+
+  .text {
+    display: flex;
+    flex-wrap: wrap;
+    width: auto;
+    height: 485px;
+  }
+
+  .text__wrap_main {
+    width: 45.5%;
+    margin: 0;
+    padding-left: 33px;
+  }
+
+  .text__wrap_title {
+    margin-bottom: 17px;
+    padding-top: 54px;
+  }
+
+  .text__content {
+    width: 90%;
+    height: 220px;
+    margin: 0;
+    padding-left: 2px;
+    letter-spacing: .6px;
+  }
+
+  .text__wrap_character {
+    flex-direction: column;
+    margin: 0;
+    padding-top: 54px;
+    padding-left: 69px;
+  }
+
+  .text__wrap_character-up {
+    height: 248px;
+  }
+
+  .text__item_bag,
+  .text__item_small-wifi { padding-bottom: 0; }
+
+  .text__item_small-screen,
+  .text__item_small-bag { padding: 10px 0 0 2px; }
+  .text__item_small-screen { margin-right: 20px; }
+  .text__item_small-bag {  margin-right: 25px; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)